bundle-patch/bundle.env

```diff
@@ -0,0 +1,20 @@
+# The pullspec should be image index, check if all architectures are there with: skopeo inspect --raw docker://$IMG | jq
+# The names must match the names of the Component CRs in Konflux.
+# Separate variables by empty lines due to merge conflicts
+
+TEMPO_TEMPO_IMAGE_PULLSPEC=quay.io/redhat-user-workloads/rhosdt-tenant/tempo/tempo@sha256:7ad3f3e5f32457a2f2bf79e0ceffb9f988183a8fb4654d39f5d6496ca0ae9b70
```

Comment:
Could you please document the convention to match with the component name?

Reply:
done